Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/modify.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2025 Apple Inc. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1.  Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+# 2.  Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import os
+import re
+import sys
+
+from .command import Command
+from .trace import Trace, Relationship
+from .commit import Commit as CommitProgram
+
+from webkitbugspy import Tracker, bugzilla, radar
+from webkitcorepy import arguments, run, string_utils, Terminal
+from webkitscmpy import local, log
+from ..commit import Commit
+
+
+def add_keyword_to_issue(issue, keyword):
+    if not isinstance(issue.tracker, radar.Tracker):
+        for candidate in issue.references:
+            if isinstance(candidate.tracker, radar.Tracker):
+                issue = candidate
+                break
+
+    try:
+        issue.set_keywords([keyword] + issue.keywords)
+        log.info(f"Successfully added '{keyword}' to the issue {issue}")
+    except radar.radarclient().exceptions.UnsuccessfulResponseException:
```

Comment:
Can you test this without `radarclient()` available? I think this would result in an exception. Need an early exist at the beginning of the function